src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -349,5 +354,21 @@ void GcsPlacementGroupManager::OnNodeDead(const NodeID &node_id) {
   SchedulePendingPlacementGroups();
 }
 
+void GcsPlacementGroupManager::Tick() const {
+  UpdatePlacementGroupLoad();
+  execute_after(io_context_, [this] { Tick(); }, 1000 /* milliseconds */);
```

Comment:
We currently include pg load to heartbeat only 1 per seconds. Lmk if you'd like to fix this.

Reply:
This is probably fine. For reference, I believe the rest of the heartbeats are updated every 100ms. 

@AmeerHajAli what do you think?